meetings/2024-09-27.md

```diff
@@ -0,0 +1,161 @@
+* HO(Hiroshi Ota): Just started in this community. 
+
+
+
+### Solid CG Charter
+URL: https://www.w3.org/community/solid/charter/
+
+#### CG Work Items
+URL: https://solidproject.org/TR/
+
+* VB: Noting Out of Scope section in the CG's charter: "Community Group’s Work Items that have transitioned to a deliverable of an active Working Group." The Solid CG will coordinate with LWS WG and pause work on such items, and may continue as needed after the WG which is [chartered until 8 September 2026](https://www.w3.org/2024/09/linked-web-storage-wg-charter.html). Such work would typically include further incubation of applicable works; documenting erratas, new CG report versions.
+* PAC: What the LWS charter says is that the primary input documents for working deliverables, Solid Protocol. The CG should probably refrain from making changes this one now. In custody of the WG so to speak.
+* PAC: The other inputs are depends on the WG. Depends on the dependencies of the Protocol. At the momment it is still flexible. I suppose the CG could still work on those. In fact, the WG charter says it depends on the maturity of the works. IMO, the CG should refrain on the Protocl at the moment. And we can synch on regular basis. If CG wants to transfer other groups they can and of course if WG agrees.
+* RG: When working on the new specification, are you going ot work on scratch or build on the current protocol. Because you're doing the Use Case from scratch. When new UCs emerge, it is not clear to me if you build on Solid Spec or start fresh.
+* PAC: My understanding is that this will be the WG's definition.
+* LD: See how input and output documents. There is the first meeting in two weeks time. It'd be premature to say that today.
+* eP: For the CG-WG coordination. I posted FedID transition proposal: https://github.com/w3c-fedid/Administration/blob/main/proposals-CG-WG.md . In LWS there are tentative deliverables. We could try to adapt this process.
+* SC: I want to add to PAC's point. Regarding the protocol it's a clear document, an input document to be continued with LWS. Why any of these odcumnets make it to WG is because there is significant incubation, not only ??? but implementation experience. In the past CG has been trying to behave like a WG would do. Makes the transition easier. If you look at the charters, there's emphasis on that at some poitn to get to a rec there needs to be open and interoprable implementations to show that it's not just a document. If the CG can focus more on showing that these things are real, they're used, there's good adoption, thatwill help us strengthen the argument that the work should continue in the WG or elsewhere. I'd like to separate the concerns of not only polishing documents but carrying implementations and test suites forward. If we can make the case that we have a good track record with implementations it'd help push those forward. 
```

Comment:
```suggestion
* SC: I want to add to PAC's point. Regarding the protocol it's a clear document, an input document to be continued with LWS. Why any of these documents make it to WG is because there is significant incubation, not only writing a specification but implementation experience. In the past CG has been trying to behave like a WG would do. Makes the transition easier. If you look at the charters, there's emphasis on that at some point to get to a REC there needs to be open and interoperable implementations to show that it's not just a document. If the CG can focus more on showing that these things are real, they're used, there's good adoption, that will help us strengthen the argument that the work should continue in the WG or elsewhere. I'd like to separate the concerns of not only polishing documents but carrying implementations and test suites forward. If we can make the case that we have a good track record with implementations it'd help push those forward. 
```